.github/workflows/default.yml

```diff
@@ -0,0 +1,34 @@
+name: Mimiker
+
+on: [push, pull_request]
```

Comment:
I think we need to disable `pull_request` otherwise _GH Actions_ starts whole run twice.

Reply:
done